pkg/test/pods.go

```diff
@@ -36,6 +38,15 @@ type PodOptions struct {
 	Tolerations          []v1.Toleration
 	Conditions           []v1.PodCondition
 	Annotations          map[string]string
+	Labels               map[string]string
+}
+
+type PDBOptions struct {
+	Name           string
+	Namespace      string
+	MinAvailable   *intstr.IntOrString
+	Selector       *metav1.LabelSelector
```

Comment:
I can't find the comment where I said this, but let's make this just a map[string]string